test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/FeedTypeUtilityTests.cs

```diff
@@ -46,6 +46,19 @@ public void FeedTypeUtility_VerifyBadSourceIsUnknown()
             Assert.Equal(FeedType.FileSystemUnknown, type);
         }
 
+        [Theory]
+        [InlineData("../foo/packages")]
+        [InlineData(@"..\foo\packages")]
+        [InlineData(@"packages")]
+        public void FeedTypeUtility_VerifyRelativePathIsUnknown(string source)
+        {
+            // Arrange & Act
+            var type = FeedTypeUtility.GetFeedType(new PackageSource(source));
+
+            // Assert
+            Assert.Equal(FeedType.FileSystemUnknown, type);
```

Comment:
Yup, looks good. Note that the product change in this area is only changing an `InvalidOperationException` thrown by calling `IsFile` on a relative `Uri` to another `InvalidOperationException` that has a better message.